appendix-45gs02-registers.tex

```diff
@@ -205,15 +205,13 @@ \subsubsection{VIC-III ``ROM'' Banking}
 \endlastfoot
 \small 0 & \small CRAM2K & \$1F800 -- \$1FFFF, \$FF80000 -- \$FF807FF & \$D800 -- \$DFFF & Y \\
```

Comment:
Actually, CRAM2K is only affecting the visibility of the second kb of colour ram at $DC00. $1F800 will always show 2kb of colram independent of CRAM2K. $D800-$DBFF will always show the first kb of colram if I/O is enabled (exception is the eth I/O personality which will show eth rx/tx buffer there instead).

Visibility of Colour RAM at $DC00 is also dependent on whether ethernet rx/tx buffer or sd card buffer is made visible instead.

Reply:
Corrected the original text.

My long term plan for some of the older appendix text is to rewrite conceptual and tutorial-style material into the main chapters, then make the appendices shorter and more concise with a focus on tabular reference information. (The appendices can then refer to the chapters for concepts.) In the meantime, we can continue to keep it factually accurate, so as not to confuse developers and future doc writers.